scriptler/checkNodesLauncherVersion.groovy

```diff
@@ -12,19 +12,32 @@ import hudson.remoting.Launcher
 import hudson.slaves.SlaveComputer
 import jenkins.model.Jenkins
 
-def expectedVersion = Launcher.VERSION
-println "Expected Version = ${expectedVersion}"
+def expectedAgentVersion = Launcher.VERSION
+def expectedJavaVersion = System.getProperty("java.version")
+println "Master"
+println " Expected Agent Version = '${expectedAgentVersion}'"
+println " Expected Java Version = '${expectedJavaVersion}'"
 Jenkins.instance.getComputers()
         .findAll { it instanceof SlaveComputer }
         .each { computer ->
+    println "Node '${computer.name}'"
```

Comment:
Perhaps this should just be `print` instead of `println`.  This way one node per line output shows up.  Considering the rest of the output starts with a space I assume that was your original intent.